### DISCUSSION TITLE
What is up with slow metabolism? Is it exclusively meant for buff afflictions now?

### DISCUSSION THREAD

**Discussion by Oukido** [FakeFishGames/Barotrauma#8048] [Ideas and suggestions]
When I was playing with a Mechanic who had a lot of max health boost I began noticing that tonic liquid was way weaker than it used to be. So I did some testing and it seems not affect medical treatments anymore. Although the Wiki states the following:
"Giving Tonic Liquid to a critically wounded patient may be worth it so as to increase the efficiency of further treatments. " [-wiki](https://barotraumagame.com/wiki/Buffs)

In the past this seems to have been true [ticket #2327](https://github.com/Regalis11/Barotrauma/issues/2327) but now the only "treatment" that seems to actually be influenced by it is the combat stimulant. (And chem boosters which I do not count as treatment here)

Also since we have alot of Health boosts in the game now, a big chunk of builds is affected by the diminishing returns of high vitality which seem to reduce the buff-affliction gain [-M61Dev](https://github.com/Regalis11/Barotrauma/issues/7105#issuecomment-957139929)

I think it suffers too much of the limitations stated above to be used as a supplement for other medications, besides the mild healing effect it provides over time (which in itself is ok)

Any Thoughts?

**Comment by Terragen** [comment]
Slow Metabolism grants a buff duration multiplier from 1x-2x scaled to strength (min 0, max 600) that affects all afflictions with the isbuff flag set to true.

Afflictions like:
- Hyperactivity
- Husk Infection Resistance
- Psychosis Resistance
- Pressure Resistance
- Vigor
- Paralysis Resistance
- and Slow Metabolism itself.


https://github.com/Regalis11/Barotrauma/blob/master/Barotrauma/BarotraumaShared/SharedSource/Characters/Health/Buffs/BuffDurationIncrease.cs#L33
